.gitignore

```diff
@@ -9,3 +12,5 @@ hack/config-provider-*
 cluster/.console.vv
 build-tools/desc/desc
 hack/config-local.sh
+
+*.swp
```

Comment:
is this created during the build?

Reply:
No, it's created by Vim editor when file is being edited. It's not strictly related to the PR but whenever I have an editor open in the tree, I get "-dirty" suffix in /.version because of that.

Reply:
You should use global gitignore ;) https://help.github.com/articles/ignoring-files/#create-a-global-gitignore